src/network/error-parsing.js

```diff
@@ -137,6 +138,10 @@ function _treatEntityExceptions(responseJSON = {}, options) {
     }else {
         fieldErrors = fieldJSONError;
     }
+
+  	for (var field in fieldErrors) {
+  		fieldErrors[field] =  translate(fieldErrors[field]);
+    }
```

Comment:
or `Object.keys(fieldErrors).reduce((res, field)=>{res[field] = translate(fieldErrors[field]); return res;}, {})` to avoid object lutation but its fine